### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by JenFau** [zsviczian/excalibrain#198]
I was working in a vault when excalibrain stopped showing anything in its graph and instead shows the 'to get started message'. Trying to search for a document on the top left hand side of the graph does not change that message although the breadcrumbs of the note do appear along the bottom. Clicking on the breadcrumbs changed the note in the note view, but excalibrain remained empty. 

Tried without success
- Restarting obsidian
- Waiting for twenty minutes in case it is a re-indexing issue
- Deleting the excalibrain.md file
- Uninstalling and reinstalling the plug in. 

I then created a new vault and installed the plug ins, which worked well. However, copying over the notes to the new vault resulted in the same issue.

**Comment by zsviczian** [comment]
Can you share a screenshot of your developer console CTRL+SHIFT+i or CMD+OPT+i

Can you remember what metadata or property you recently added to your vault. The issue is with some of the document property handling, the problem is I haven't been able to find it myself as it does not happen in my Vault, but have heard it from others.

**Comment by JenFau** [comment]
Oh! I think you've provided the answer. I'd typed in a property of Note type - renaming this to note_type solved the problem. Thank you.

**Comment by JenFau** [comment]
Although if I add note-type to the exclusion box, it still shows up in my graph. Is that expected behaviour, please?

**Comment by zsviczian** [comment]
yes, exclusion is a misleading name, it is excluded from the suggester. I've now added a new category: hidden, will release soon.

**Comment by JenFau** [comment]
Great, thank you so much.
